### ISSUE TITLE
Doesn't work on recent versions of MySQL due to uppercase columns in response

### ISSUE THREAD

**Issue by lumosmind** [TehShrike/joi-sql#5]
Win 10
node v10.15.1
database: cloud mysql

I can get table information via this snippet
```
const table_name = "users"
const sql = `desc ${table_name}`;
con.query(sql, function (err, resuts) {
  if (err) throw err;
  console.log(resuts);
  con.end();
});
```

But joi- sql returns:

```
Joi.object({
        undefined: Joi,
        undefined: Joi,
        undefined: Joi,
        undefined: Joi,
        undefined: Joi
})
```

I used joi-sql via this snippet with/without **connection: con** parameter
```
const con = mysql.createConnection({
  host: '***********',
  user: '*******',
  password: '************',
  database: '**********'
});


joiSql({
  host: '**********************',
  user: '*******',
  password: '********',
  schema: '*********',
  table: '****',
  camel: false,
  connection: con
}).then(result => {
  // typeof result // => 'string'
  console.log(result);
})
```

**Comment by TehShrike** [comment]
What version of `mysql`?

What do you see if you add `console.log(columns)` here? https://github.com/TehShrike/joi-sql/blob/bb86f75ab2caaddf263e3a7dfce643e4002af228/index.js#L24

**Comment by lumosmind** [comment]
mysql version : 8.0.13-3

I get output after adding this  `console.log(columns)` 

```js
[ { COLUMNNAME: 'email',
    DATATYPE: 'varchar',
    COLUMNTYPE: 'varchar(255)',
    NUMERICPRECISION: null,
    NUMERICSCALE: null,
    CHARACTERMAXIMUMLENGTH: 255,
    ISNULLABLE: 'YES' },
  { COLUMNNAME: 'id',
    DATATYPE: 'int',
    COLUMNTYPE: 'int(11)',
    NUMERICPRECISION: 10,
    NUMERICSCALE: 0,
    CHARACTERMAXIMUMLENGTH: null,
    ISNULLABLE: 'NO' },
  { COLUMNNAME: 'iseMailValid',
    DATATYPE: 'tinyint',
    COLUMNTYPE: 'tinyint(1)',
    NUMERICPRECISION: 3,
    NUMERICSCALE: 0,
    CHARACTERMAXIMUMLENGTH: null,
    ISNULLABLE: 'YES' },
  { COLUMNNAME: 'name',
    DATATYPE: 'varchar',
    COLUMNTYPE: 'varchar(255)',
    NUMERICPRECISION: null,
    NUMERICSCALE: null,
    CHARACTERMAXIMUMLENGTH: 255,
    ISNULLABLE: 'YES' },
  { COLUMNNAME: 'password',
    DATATYPE: 'varchar',
    COLUMNTYPE: 'varchar(255)',
    NUMERICPRECISION: null,
    NUMERICSCALE: null,
    CHARACTERMAXIMUMLENGTH: 255,
    ISNULLABLE: 'YES' } ]
```

```js
Joi.object({
        undefined: Joi,
        undefined: Joi,
        undefined: Joi,
        undefined: Joi,
        undefined: Joi
})
```

**Comment by TehShrike** [comment]
hmm, I'm not sure how that would happen, the `fetch` code is calling `camelize` on the output before returning it: https://github.com/TehShrike/joi-sql/blob/master/fetch.js#L20

**Comment by TehShrike** [comment]
ah, I think `camelize` doesn't handle uppercased inputs well.

Switching to https://github.com/sindresorhus/camelcase-keys/ should fix it

**Comment by ZeviLiao** [comment]
I've fixed my local about the uppercase keywords
please reference the fork build.js file
https://github.com/ZeviLiao/joi-sql/blob/master/build.js

**Comment by TehShrike** [comment]
That fix doesn't work for everyone.

If anyone wants to fix the problem for all versions of MySQL, a PR that changes the test to include an UPPER_SNAKE_CASE column in the response, and switches from `camelize` to `camelcase-keys` would be appreciated.

**Comment by TehShrike** [comment]
Fix published as 1.4.1
